### DISCUSSION TITLE
Is it possible to compare a text file (saved) and text content (not saved to file)

### DISCUSSION THREAD

**Discussion by rbharathkumar** [WinMerge/winmerge#2236] [Q&A] [answered]
Hi All,
Just wanted to understand if it is possible to perform blank comparison between a text file and some typed/copy pasted text (not saved to a file).

Example,
Assume a file `a.txt` having below content.
```
1
2
3
```

Assume we just have some text - not saved to any file.
```
4
5
6
```

Basically looking for a way to compare a saved file with some typed or copy pasted content.
In Meld, we used to perform the same as below. 
Select Blank Comparison -> Paste content on Left Pane -> Open saved file on Right Pane -> Compare

(Mentioning meld only to showcase the process flow, not for comparing apps and features)

Thank you for all the help.
Cheers...

**Comment by rbharathkumar** [comment]
Figured it out myself. For future use.

New Document -> On Left Pane -> On the right Click and Select Open -> Select File

![image](https://github.com/WinMerge/winmerge/assets/38585383/ad55f1b5-70a0-468c-9080-ca842bfab49b)

On Right Pane, type or copy paste content -> Click Refresh.

![image](https://github.com/WinMerge/winmerge/assets/38585383/aa5b4623-f0d8-41a7-b02d-989a1c890064)

**Comment by sdottaka** [comment, marked as answer]
In WinMerge, please do as follows.

1. Press Ctrl+N or click the leftmost icon on the toolbar.
2. Click the hamburger menu in the header bar of the left pane.
3. Click the Open menu item and select the file.
4. Paste or type your text in the left pane.
5. Press F5 or click the icon on the far right of the toolbar.

![image](https://github.com/WinMerge/winmerge/assets/98126/704b17d9-4f81-4144-a7f0-a978267f8c23)

**Reply by rbharathkumar** [reply]
Thank you for your kind response...

**Comment by sdottaka** [comment]
Oh, it looks like my answer wasn't necessary.

**Reply by rbharathkumar** [reply]
Your answer is still much more clear...Appreciate your time...
